packages/design-picker/src/components/style-variation-badges/badge.tsx

```diff
@@ -18,7 +21,21 @@ const Badge: React.FC< BadgeProps > = ( { variation } ) => {
 	}
 
 	return (
-		<div className="style-variation__badge-wrapper">
+		<div
+			className="style-variation__badge-wrapper"
+			tabIndex={ 0 }
+			role="button"
+			onClick={ ( e ) => {
+				e.stopPropagation();
```

Comment:
Could you add the comment to explain why we need to stop propagation? thanks!

Besides, we might need to show the hover/active style on the current variation. Otherwise, the user might not notice the behavior of clicking on the variation is different from the entire design

Reply:
@SaxonF need your opinion on the hover/active style 🙂
Currently, I've copied the one from the theme card, but I'm not sure if that makes sense:


https://user-images.githubusercontent.com/797888/189634048-2e71114d-8f6b-433e-9ca4-5bd6c1be37c1.mp4

Reply:
@taipeicoder That's a good starting point! Maybe we can increase the opacity slightly so its a tiny bit more visible when hovering.